services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/api/rest/_resource_tracker.py

```diff
@@ -78,6 +81,38 @@ async def list_usage_services(
     )
 
 
+@router.get(
+    "/services/-/usages/presigned-link",
+    response_model=AnyUrl,
+    operation_id="get_usages_presigned_link",
+    description="Return presigned-link",
+    tags=["usages"],
+)
+async def get_usages_presigned_link(
+    request: Request,
+    user_id: UserID,
+    product_name: ProductName,
+    resource_tracker_repo: Annotated[
+        ResourceTrackerRepository, Depends(get_repository(ResourceTrackerRepository))
+    ],
+    wallet_id: Annotated[WalletID | None, Query()] = None,
+    access_all_wallet_usage: Annotated[bool, Query()] = False,
+):
+    app_settings: ApplicationSettings = request.app.state.settings
+    s3_settings = app_settings.RESOURCE_USAGE_TRACKER_S3
+    assert s3_settings  # nosec
+
+    return await resource_tracker_service_runs.export_service_runs(
```

Comment:
Q: how long does this endpoint run for?
If you need to export 3 million rows? do you get a link immediately or do you wait for the rows to be generated before providing the link?

Reply:
If you need to wait we have a problem here. I would expose this one via RPC.

Reply:
As discussed first version was requested to be easy also for the frontend (not expecting huge export files).